### PR TITLE
fix: auth modal ui

### PR DESCRIPTION
**Pull request by rohitdasu**
### Changes proposed

fixed ui issues in login modal / auth modal
* padding and alignment stuff

### Screenshots
Screenshot 2022-09-19 at 11 19 52 AM

Screenshot 2022-09-19 at 11 18 56 AM

### CODE REVIEW

File: components/AuthModal/AuthModal.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Fragment } from 'react';
 import { useAppSelector, useAppDispatch } from '../../app/hooks';
 import { closeModal } from '../../store/slices/sliceModal';
 import { signInWithPopup, signOut } from 'firebase/auth';
-import { auth, Provider_github, Provider_google } from '../../lib/firebase';
+import { auth, providerGithub, providerGoogle } from '../../lib/firebase';
```
Comment (leewan-09) [inline components/AuthModal/AuthModal.tsx:6]: `@/lib/firebase`

Comment (rohitdasu) [reply components/AuthModal/AuthModal.tsx]: done

**Review by leewan-09 — APPROVED** [review]
LGTM👍

**Review by AdityaPainuli — APPROVED** [review]
The Design looks good to me ..

**Pull request merged**